connector/src/test/scala/com/basho/riak/spark/rdd/timeseries/TimeSeriesPartitioningTest.scala

```diff
@@ -0,0 +1,153 @@
+/**
+ * *****************************************************************************
+ * Copyright (c) 2016 IBM Corp.
+ *
+ * Created by Basho Technologies for IBM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * *****************************************************************************
+ */
+package com.basho.riak.spark.rdd.timeseries
+
+import java.sql.Timestamp
+import scala.collection.JavaConversions.asScalaBuffer
+import org.apache.spark.sql.sources._
+import org.apache.spark.sql.types._
+import org.junit.Test
+import com.basho.riak.client.core.util.HostAndPort
+import com.basho.riak.spark.rdd.ReadConf
+import com.basho.riak.spark.rdd.partitioner.RangedRiakTSPartitioner
+import com.basho.riak.spark.rdd.partitioner.RiakTSPartition
+import junit.framework.Assert.assertEquals
+import org.apache.spark.SparkConf
+import org.apache.spark.SparkContext
+import org.junit.After
+
+class TimeSeriesPartitioningTest {
```

Comment:
Lets add a test to check and fix the behavior for the case when not supported Filter is used